api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -437,6 +456,20 @@ public void getActiveVisitsByPatient_shouldReturnAllUnvoidedVisitsForTheSpecifie
 		assertEquals(8, visitService.getVisitsByPatient(new Patient(2), true, false).size());
 	}
 	
+	@Test
+	public void getActiveVisitsByPatient_shouldReturnEmptyListGivenNull() {
+		assertThat(visitService.getVisitsByPatient(null, false, true), is(empty()));
+	}
+	
+	@Test
+	public void getActiveVisitsByPatient_shouldReturnEmptyVisitsForNullPatientId() {
```

Comment:
you don't test `getActiveVisitsByPatient` so the test methods should not be named after that method.

rename to `getVisitsByPatient_shouldReturnEmptyListGivenPatientWithNullPatientIdAndFalse`
(you can set both boolean parameters to false)